grammar.js

```diff
@@ -5,27 +5,31 @@
 // but may be too verbose for a syntax tree?
 // -> use fields? field('fieldname', $._hidden_token)
 
+// Values taken from lalrpop grammar. In lalrpop the highest precedence is 0.
```

Comment:
please add link to https://github.com/tweag/nickel/blob/master/src/grammar.lalrpop